src/components/modals/AddDulModal.js

```diff
@@ -57,87 +107,253 @@ export const AddDulModal = hh(class AddDulModal extends Component {
     }
   }
 
+  async uploadFile() {
+    const response = await Consent.CreateDulResource(this.state.consent.consentId, this.state.file.name, this.state.file);
+    if (response !== true) {
+      this.setState(prev => {
+        prev.error.title = 'Server Error';
+        prev.error.msg = 'Problem with the file UpLoad.';
+        prev.error.show = true;
+        return prev;
+      });
+    }
+  }
+
+  async consentHandler() {
+    const consent = {};
+    let response;
+    consent.consentId = this.state.consent.consentId;
+    consent.useRestriction = this.state.consent.useRestriction;
+    consent.dataUse = this.state.consent.dataUse;
+    consent.name = this.state.consent.name;
+    if (this.state.isEditMode) {
+      response = await Consent.update(consent);
+    } else {
+      response = await Consent.CreateConsentResource(consent);
+    }
+    if (response === true) {
+      return true;
+    } else {
+      this.handleErrors(response);
+    }
+    return false;
+  };
+
+
+  handleErrors(message) {
+    var tle = 'Conflicts to resolve!';
```

Comment:
what tle means ?

Reply:
This stands for Title related to the Error alert title. I've change this to a better variable name.